### PR TITLE
Feat(CIV-607): Added stripo email logo on the footer of the website

### PR DESCRIPTION
**Pull request by Skt2021**
## JIRA Ticket

[CIV-607](https://commutatus.atlassian.net/browse/CIV-607)

## What?
Added **Stripo's logo** to the website's footer.

## Why?
Civis has started using Stripo's services therefore the logo needs to be added to the site.

### CODE REVIEW

File: src/app/shared/components/footer/footer.component.html [hunk 1]
```diff
@@ -59,6 +59,14 @@
           all content is licensed under' | translate}}{{'a creative commons Attibution-No Derivates 4.0 International License' | translate}}
         </div>
       </div>
+      <div class="stripo-logo-container">
+        <p class="title-text">Newsletter Powered by:</p>
+        <picture>
+          <source srcset="assets/images/stripo-email.webp" type="image/webp">
+          <source srcset="assets/images/stripo-email.jpg" type="image/jpeg">
+          <img src="assets/images/stripo-email.jpg" alt="stripo email logo" />
```
Comment (gpalsingh) [inline src/app/shared/components/footer/footer.component.html:67]: Not sure what's being done here as the uploaded image format is `png` but the code is referring to a `jpg` image

File: src/app/shared/components/footer/footer.component.scss [hunk 2]
```diff
@@ -21,6 +21,7 @@
 
     .left-item {
       display: flex;
+      width: 345px;
```
Comment (gpalsingh) [inline src/app/shared/components/footer/footer.component.scss:24]: Is it possible to avoid using absolute widths?

**Review by gpalsingh — CHANGES_REQUESTED** [review]

File: src/app/shared/components/footer/footer.component.html (hunk 1 above)
Comment (Skt2021) [reply src/app/shared/components/footer/footer.component.html]: Fixed.

File: src/app/shared/components/footer/footer.component.scss (hunk 2 above)
Comment (Skt2021) [reply src/app/shared/components/footer/footer.component.scss]: Changed it to percentages.

File: src/app/shared/components/footer/footer.component.scss [hunk 3]
```diff
@@ -21,16 +21,17 @@
 
     .left-item {
       display: flex;
+      width: 27%;
```
Comment (gpalsingh) [inline src/app/shared/components/footer/footer.component.scss:24]: As discussed, let's use grid layout instead

**Review by gpalsingh — CHANGES_REQUESTED** [review]

File: src/app/shared/components/footer/footer.component.scss (hunk 3 above)
Comment (Skt2021) [reply src/app/shared/components/footer/footer.component.scss]: Done

**Review by gpalsingh — APPROVED** [review]
👏🏽

**Pull request merged**